src/cli/typings.js

```diff
@@ -33,7 +33,7 @@ const readProto = protoPath => R.pipeWith(then, [
   readdir,
   mapAsync(async fileName => path.resolve(protoPath, fileName)),
   filterAsync(isFile),
-  mapAsync(R.flip(readFile)('utf8')),
+  R.flip(readFile)('utf8'),
```

Comment:
I didn't notice your PR before, I already pushed the fix. Good catch! :)

Here the filter function will return a list (of promises) so we need to map over it to read each file. Although `R.flip` didn't work correctly with `readFile` from `fs-extra`.

Closing this PR and for the future I'll be more careful to check if there is already proposed solution. Thanks!